### PR TITLE
Added camera state

### PR DESCRIPTION
**Pull request by trym-b**
Added tests for:
    available state
    connected state
    live state
    temperature
    lens temperature
    dmd temperature
    general temperature
    led temperature
    pcb temperature

### CODE REVIEW

File: zivid/camera_state.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import _zivid
+from zivid.errors import ZividError
+
+
+class CameraState:
```
Comment (nedrebo) [inline zivid/camera_state.py]: CameraState and Settings are identical in structure. Why did you wrap this differently than Settings?

Comment (trym-b) [reply zivid/camera_state.py]: Are you refering to the property decorators?
assigning temperature might seem like the user can set the temperature on the camera, which is impossible.

Comment (nedrebo) [reply zivid/camera_state.py]: Settings code have lots of: "abc =_zivid.Settings().x.x.abc.value" I expected the same here.

Also this code does not know that it is a property of camera. In standalone mode this class should be "normal".

In the end all the "data model" classes should be identical.

The camera.state property should of course be a read only property.

**Comment by trym-b** [comment]
ready for next round

**Comment by trym-b** [comment]
Ready for next round

**Pull request merged**